remoteappmanager/application.py

```diff
@@ -205,27 +205,13 @@ def _reverse_proxy_init(self):
 
     def _db_init(self):
         """Initializes the database connection."""
-        self.db = orm.Database(self.file_config.db_url)
+        self.db = orm.AccountingInterface(self.file_config.db_url)
```

Comment:
then change the namespace...
orm.Database is clearly a sqlalchemy oriented database, and implements the Database interface with an implementation that is sqlalchemy oriented.
You will have another implementation that handles csv files. so you need a db namespace, and two db backends: sqlalchemy and csv.